Fireworks/Core/src/FWGUIEventDataAdder.h

```diff
@@ -22,6 +22,7 @@
 #include <set>
 #include <vector>
 #include <string>
+#include <TGFrame.h>
```

Comment:
@mrodozov , isn't forward declaration good enough in this case?

Reply:
It is pr is modified.